libraries/Filter/LowPassFilter.h

```diff
@@ -15,22 +15,20 @@
 
 //
 /// @file	LowPassFilter.h
-/// @brief	A class to implement a low pass filter without losing precision even for int types
-///         the downside being that it's a little slower as it internally uses a float
-///         and it consumes an extra 4 bytes of memory to hold the constant gain
+/// @brief	A class to implement a low pass filter.
 
 /*
-  Note that this filter can be used in 2 ways:
+  Two classes are provided:
 
-   1) providing dt on every sample, and calling apply like this:
+   LowPassFilter: providing dt on every sample, and calling apply like this:
 
       // call once
       filter.set_cutoff_frequency(frequency_hz);
 
       // then on each sample
       output = filter.apply(sample, dt);
 
-   2) providing a sample freq and cutoff_freq once at start
+   LowPassFilter_const_dt: providing a sample freq and cutoff_freq once at start
```

Comment:
I'm really not enamoured with the name!

Reply:
I'm open to better suggestions, I just wanted to make it clear which should be used in each case.

Reply:
FixedLowPassFilter? StaticLowPassFilter?

Reply:
Its not fixed in cutoff frequency, its fixed in sample rate. So i'm not sure fixed really works. Or at least it does not line up with what we would call a fixed notch.

Reply:
LowPassFilterConstDt would be my vote